integration/tests/basic/erc/test_ERC173.py

```diff
@@ -0,0 +1,34 @@
+import allure
+import pytest
+import web3
+
+from integration.tests.basic.helpers.basic import BaseMixin
+
+
+@allure.feature("ERC Verifications")
+@allure.story(
+    "ERC-173: Contract Ownership Standard"
+)
+class TestERC173ContractOwnershipStandard(BaseMixin):
```

Comment:
Standard has event on owner change, please check it too. 

> with a standardized event for when ownership is changed (OwnershipTransferred(address indexed previousOwner, address indexed newOwner)

And case:

> To renounce any ownership of a contract set _newOwner to the zero address: transferOwnership(address(0)). If this is done then a contract is no longer owned by anybody.

And check this calling "changeOwner" method from contract will be good too